source/common/common/hash.h

```diff
@@ -25,6 +25,24 @@ class HashUtil {
     return XXH64(input.data(), input.size(), seed);
   }
 
+  /**
+   * Return 64-bit hash from the xxHash algorithm.
+   * @param input supplies the absl::InlinedVector<absl::string_view> to hash.
+   * @param seed supplies the hash seed which defaults to 0.
+   * See https://github.com/Cyan4973/xxHash for details.
+   */
+  static uint64_t xxHash64(absl::InlinedVector<absl::string_view, 1>& input, uint64_t seed = 0) {
```

Comment:
Move the body to to the .cc file.

Reply:
ok, got it

Reply:
I would recommend passing an absl::Span here so it can take any type of container, and also making it const.